### PR TITLE
Monitor fixes

### PR DESCRIPTION
**Pull request by eregon**
For CI.

### CODE REVIEW

**Comment by eregon** [comment]
(This was rebased and pushed tp master)

File: truffleruby/src/main/java/org/truffleruby/Main.java [hunk 1]
```diff
@@ -139,19 +139,15 @@ public static void main(String[] args) {
                         Log.performanceOnce("this JVM does not have the Graal compiler - performance will be limited - see doc/user/using-graalvm.md");
                     }
 
+                    final String bootCode;
                     if (config.shouldUsePathScript()) {
                         context.setOriginalInputFile(config.getScriptFileName());
-                        exitCode = engine.eval(loadBootSource(
-                                //language=ruby
```
Comment (pitr-ch) [inline truffleruby/src/main/java/org/truffleruby/Main.java]: Please do not remove these comments, intellij is able to index to Ruby code with them in the Strings.

Comment (eregon) [inline truffleruby/src/main/java/org/truffleruby/Main.java]: Right, could you add them to the new String literals? I can't easily test with IntelliJ.
How does it help? Does it notice there is a caller of Truffle::Boot.main then?

Comment (pitr-ch) [inline truffleruby/src/main/java/org/truffleruby/Main.java]: Yeah I am planning to add it, I could have made that clear. Yeah it behaves as any other Ruby code, so navigation to definition and find usages works as expected.

**Pull request closed without merging**